### ISSUE TITLE
Frequent error when interacting with console and other windows

### ISSUE THREAD

**Issue by CatalinaYepes** [spyder-ide/spyder#10050]
### What steps will reproduce the problem?

The problem occurs when interacting with the cursor between the console and the Variable explorer option.

### Traceback
```python-traceback
  File "/Users/cyepes/py36/lib/python3.6/site-packages/qtconsole/completion_widget.py", line 83, in hideEvent
    self._text_edit.cursorPositionChanged.disconnect(self._update_current)
TypeError: 'method' object is not connected
```

## Versions

* Spyder version: 3.3.6 
* Python version: 3.6.8
* Qt version: 5.12.2
* PyQt5 version: 5.12.1
* Operating System: Darwin 18.7.0

### Dependencies

```
pyflakes >=0.6.0  :  1.6.0 (OK)
pycodestyle >=2.3 :  2.3.1 (OK)
pygments >=2.0    :  2.3.1 (OK)
sphinx >=0.6.6    :  1.8.5 (OK)
rope >=0.9.4      :  0.12.0 (OK)
jedi >=0.9.0      :  0.13.3 (OK)
psutil >=0.3      :  5.4.3 (OK)
nbconvert >=4.0   :  5.4.1 (OK)
pandas >=0.13.1   :  0.24.1 (OK)
numpy >=1.7       :  1.14.2 (OK)
sympy >=0.7.3     :  None (NOK)
cython >=0.21     :  None (NOK)
qtconsole >=4.2.0 :  4.4.3 (OK)
IPython >=4.0     :  7.3.0 (OK)
matplotlib >=2.0.0:  2.1.2 (OK)
pylint >=0.25     :  2.3.1 (OK)
```

**Comment by ccordoba12** [comment]
Please update the `qtconsole` package to its 4.5.3 version and try again. If you used pip to install Spyder, please use the following command in Terminal.app:

    pip install -U qtconsole

With Anaconda, the command you need to execute is

    conda update qtconsole

**Comment by impact27** [comment]
I can reproduce with:
In the console:
1. `abba = 1`
2. `ab<tab>`
3. A widget with `abs` and `abba` appears
4. Click out of the widget
5. 
```python-traceback
Traceback (most recent call last):
  File "/Users/quentinpeter/pyscripts/qtconsole/qtconsole/completion_widget.py", line 86, in hideEvent
    self._text_edit.cursorPositionChanged.disconnect(self._update_current)
TypeError: 'method' object is not connected
```

**Comment by ccordoba12** [comment]
Thanks for the confirmation @impact27!

@CatalinaYepes, the fix will be available in the next qtconsole version (**4.5.4**). If you use conda, you'll need to wait until next week and then run

    conda update qtconsole

With pip you can run

    pip install -U qtconsole

tomorrow.

Finally, please **do not** use pip to update this package if you use conda because it'll break your installation.

**Comment by CatalinaYepes** [comment]
Thank you!
